include/rdma/fi_endpoint.h

```diff
@@ -60,6 +60,7 @@ enum {
 enum {
 	FI_OPT_MIN_MULTI_RECV,		/* size_t */
 	FI_OPT_CM_DATA_SIZE,		/* size_t */
+	FI_OPT_BUFFERED_MIN,		/* size_t */
```

Comment:
This must be placed at the end of the enum or it breaks the ABI

Reply:
Should the previously added `FI_OPT_BUFFERED_LIMIT` be also moved down?

Reply:
Looking at the 1.6.x branch, rhis change is actually fine.  Only MIN_MULTI_RECV and CM_DATA_SIZE values have been part of a release.  Everything after that is new for 1.7, so the order doesn't matter yet.